### DISCUSSION TITLE
How to run HTTPS locally?

### DISCUSSION THREAD

**Discussion by guoyunhe** [vercel/next.js#16082] [Help]
So I am developing a Next.js app with Firebase. Firebase Messaging requires HTTPS to work. How could I run Next.js app in HTTPS?

**Comment by runoncedev** [comment]
There is an old discussion regarding this:

https://github.com/vercel/next.js/discussions/10935

Even if it's not a built-in feature I wish Next.js docs would include some sort of an official guide on how to propertly set up a local environment with HTTPS.
